mysql/check.py

```diff
@@ -325,14 +319,15 @@ def _get_config(self, instance):
         self.mysql_sock = instance.get('sock', '')
         self.defaults_file = instance.get('defaults_file', '')
         user = instance.get('user', '')
-        password = instance.get('pass', '')
+        password = str(instance.get('pass', ''))
         tags = instance.get('tags', [])
         options = instance.get('options', {})
         queries = instance.get('queries', [])
         ssl = instance.get('ssl', {})
+        connect_timeout = instance.get('connect_timeout', None)
```

Comment:
there should be an update on the yaml example file to add the `connect_timeout` parameter